tools/protoprint/protoprint_test.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python3
+
+import argparse
+import logging
+import os
+import pathlib
+import re
+import subprocess
+import sys
+import tarfile
+import tempfile
+
+from tools.protoprint.test_data import data as test_data
+from tools.run_command import run_command
+
+from envoy_repo import PATH
+
+
+def path_and_filename(label):
+    """Retrieve actual path and filename from bazel label
+
+    Args:
+        label: bazel label to specify target proto.
+
+    Returns:
+        actual path and filename
+    """
+    if label.startswith('/'):
+        label = label.replace('//', '/', 1)
+    elif label.startswith('@'):
+        label = re.sub(r'@.*/', '/', label)
+    else:
+        return label
+    label = label.replace(":", "/")
+    splitted_label = label.split('/')
+    return '/'.join(splitted_label[:len(splitted_label) - 1])[1:], splitted_label[-1]
```

Comment:
TBH this is all a bit magical. I think it was before (in the aspect / Bazel rule logic), but still is. Can you include some example paths to show what this is doing?

Reply:
sure - again its just what it was doing previously, but copied out (with exception that it now takes off first char)